qa/common/shm_util.py

```diff
@@ -53,12 +53,19 @@ def create_register_shm_regions(input0_list, input1_list, output0_byte_size,
             "Cannot set both System and CUDA shared memory flags to 1")
 
     if not (use_system_shared_memory or use_cuda_shared_memory):
-        return []
+        return [], None, None
 
     triton_client = httpclient.InferenceServerClient("localhost:8000")
```

Comment:
Same here. Can you pass the protocol to "create_register_shm_regions" and then use appropriate client?

Reply:
Similarly, created once for all configs and reused after that